docs/tables/oci_core_volume.md

```diff
@@ -0,0 +1,62 @@
+# Table: oci_core_volume
+
+The Oracle Cloud Infrastructure Block Volume service lets you dynamically provision and manage block storage volume
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  id,
```

Comment:
select id > should we mention an alias for provide some info e.g. select id as volume_id.